data/json/items/melee.json

```diff
@@ -2258,5 +2258,24 @@
     "techniques": "RAPID",
     "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 7 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
+  },
+  {
+    "type": "TOOL",
+    "category": "weapons",
+    "id": "grip_hook",
+    "name": "grip hook",
+    "description": "A simple steel meathook with a simple steel handle. Makes a passable melee weapon.",
```

Comment:
```suggestion
    "description": "A simple steel meathook with a simple steel handle.  Makes a passable melee weapon.",
```

Reply:
It's a draft PR which I will be working on for a while. I thought I'd start it here, but it's nowhere near finished, including the description of the PR

Reply:
I can understand how from a maintainer however that a PR without a good description is difficult to parse, I'll make a decent description as my next step.